src/rpcgovernance.cpp

```diff
@@ -190,6 +190,9 @@ UniValue gobject(const UniValue& params, bool fHelp)
             govobj.SetMasternodeInfo(mn.vin, activeMasternode.pubKeyMasternode);
             govobj.Sign(activeMasternode.keyMasternode);
         }
+        else {
```

Comment:
Won't this lead to situation where all non-`GOVERNANCE_OBJECT_TRIGGER` objects always throw an exception here?